test/forkMainnet/AMMQuoter.t.sol

```diff
@@ -12,14 +12,14 @@ contract AMMQuoterTest is StrategySharedSetup {
 
     AMMQuoter ammQuoter;
 
-    address DEFAULT_MAKER_ADDR = UNISWAP_V2_ADDRESS;
-    address DEFAULT_TAKER_ASSET_ADDR = DAI_ADDRESS;
-    address DEFAULT_MAKER_ASSET_ADDR = USDT_ADDRESS;
+    address DEFAULT_MAKER_ADDR;
+    address DEFAULT_TAKER_ASSET_ADDR;
+    address DEFAULT_MAKER_ASSET_ADDR;
```

Comment:
Can not assign default value to address variables here as the addresses do not have default values.

Reply:
Oh I didn't realize that addresses can only be used after constructor called 🥲 But if we keep previous version, it will always get the mainnet addresses no matter in which network, since the override happens after constructor called, right?

Reply:
Yes, it will always default to mainnet addresses.

> Oh I didn't realize that addresses can only be used after constructor called 🥲

Me neither until I was writing it...😂😅